targets/roslyn/project.json

```diff
@@ -0,0 +1,26 @@
+﻿{
+  "runtimes": {
+    "win7-x86": {},
+    "win7-x64": {},
+    "osx.10.10-x64": {},
+    "ubuntu.14.04-x64": {},
+    "ubuntu.16.04-x64": {}
+  },
+  "dependencies": {},
+  "frameworks": {
+    "net46": {
+      "dependencies": {}
```

Comment:
Super confusingly, no. This was actually related to the test failures. The problem is that the `Microsoft.Net.Compilers.netcore` package metadata delivers everything nicely as assets, so the "publish" step includes it all, but the full-framework version of the package has everything in tools instead. So we copy it an entirely different way.